### ISSUE TITLE
How fast？

### ISSUE THREAD

**Issue by haochange** [Wanggcong/Spatial-Temporal-Re-identification#1]
the speed

**Comment by Wanggcong** [comment]
As described in paper, the only difference between conventional person re-ID ans st-ReID is the spatial-temporal stream. During training, we have to estimate the spatial-temporal distribution, which takes ~1.5 minutes without using GPU. During testing, it nearly does not increase extra overhead compared with the time consumption of appearance feature extraction.

**Comment by haochange** [comment]
> As described in paper, the only difference between conventional person re-ID ans st-ReID is the spatial-temporal stream. During training, we have to estimate the spatial-temporal distribution, which takes ~1.5 minutes without using GPU. During testing, it nearly does not increase extra overhead compared with the time consumption of appearance feature extraction.

sorry to bother u. Could you provide models？During testing，how fast is it with GPU? Now,  I am training the model with default hyperparameters in the train_market.py, but the acc is not normal, so could you give me some advice? Thank you so much.  :)  @Wanggcong 

---------------------------------------------------------------------------------------

Epoch 62/599
----------
train Loss: 1.14366493007 Acc: 0.0923005565863
val Loss: 1.18377197916 Acc: 0.0492676431425

Epoch 63/599
----------
train Loss: 1.14328388646 Acc: 0.0918367346939
val Loss: 1.1837657517 Acc: 0.0545938748336

Epoch 64/599
----------
train Loss: 1.14249895057 Acc: 0.0940012368584
val Loss: 1.18305726947 Acc: 0.0532623169108

Epoch 65/599
----------
train Loss: 1.14220841578 Acc: 0.0920686456401
val Loss: 1.18462524211 Acc: 0.0545938748336

Epoch 66/599
----------
train Loss: 1.14185523486 Acc: 0.0946196660482
val Loss: 1.18472640524 Acc: 0.0572569906791

Epoch 67/599
----------
train Loss: 1.14138634174 Acc: 0.0942331478046
val Loss: 1.18505184311 Acc: 0.0585885486019

Epoch 68/599
----------
train Loss: 1.14116656257 Acc: 0.094696969697
val Loss: 1.1813824599 Acc: 0.0545938748336

Epoch 69/599
----------
train Loss: 1.14033726166 Acc: 0.0932282003711
val Loss: 1.18494988758 Acc: 0.0572569906791

Epoch 70/599
----------
train Loss: 1.13984184937 Acc: 0.0940785405071
val Loss: 1.18241847276 Acc: 0.0505992010652

Epoch 71/599
----------
train Loss: 1.13958641653 Acc: 0.0936920222635
val Loss: 1.18271717219 Acc: 0.0532623169108

**Comment by haochange** [comment]
I change the python version from 2.7 to 3.6, the pytorch verson from 0.3 to 0.4.0.
 
python3.6
pytorch0.4.0
the acc is good.
-------------------------------------------------------------------------------------------
Epoch 5/59
----------
train Loss: 0.5127 Acc: 0.6110
val Loss: 0.4457 Acc: 0.6378

Epoch 6/59
----------
train Loss: 0.4656 Acc: 0.6681
val Loss: 0.3587 Acc: 0.7217

Epoch 7/59
----------
train Loss: 0.4335 Acc: 0.7039
val Loss: 0.3176 Acc: 0.7870

Epoch 8/59
----------
train Loss: 0.4075 Acc: 0.7390
val Loss: 0.2971 Acc: 0.8149

Epoch 9/59
----------
train Loss: 0.3839 Acc: 0.7700
val Loss: 0.2578 Acc: 0.8495
